server/lambda/layer/common/nodejs/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ets-layer",
   "version": "1.0.0",
   "dependencies": {
-    "axios": "0.19.2",
+    "axios": "^0.21.1",
```

Comment:
probably want to lock this

Reply:
Lol I touched like 4 lines and still forgot something 🤦

Reply:
Fixed.